flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/gateway/local/LocalExecutor.java

```diff
@@ -240,7 +240,11 @@ public void closeSession(String sessionId) throws SqlExecutionException {
 			}
 		});
 		// Remove the session's ExecutionContext from contextMap.
-		this.contextMap.remove(sessionId);
+		try {
+			this.contextMap.remove(sessionId).close();
+		} catch (IOException e) {
+			// ignore any throwable to keep the clean up running
```

Comment:
should still be logged

Reply:
Should we then also log on cancelQuery on L237? I tried to keep it symmetric.

Reply:
I think so, as a separate commit of course ;)